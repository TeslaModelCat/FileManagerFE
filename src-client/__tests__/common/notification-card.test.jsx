import React from 'react';
import { mount } from 'enzyme';
import NotificationCard from '../../components/common/notification-card';

describe('rendering', () => {
  it('renders correctly', () => {
    const tree = mount(<NotificationCard notification={{ isRead: false, type: 'file_removed' }} />);

    expect(tree).toMatchSnapshot();
  });
});
