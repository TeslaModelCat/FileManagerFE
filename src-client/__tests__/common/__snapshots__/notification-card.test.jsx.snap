// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendering renders correctly 1`] = `
<NotificationCard
  notification={
    Object {
      "isRead": false,
      "type": "file-removed",
    }
  }
>
  <div
    className="notification-card unread"
  >
    <div
      className="message"
    >
      File removed
    </div>
    <div
      className="marker"
    />
  </div>
</NotificationCard>
`;
